packages/format-message-cli/index.js

```diff
@@ -98,6 +112,22 @@ program
         errors.push(err.message)
       }
     }
+    if (options.extends === 'default') {
+      options.extends = [ 'plugin:format-message/default' ]
+      options.customrules = null
+    } else if (options.extends === 'recommended') {
+      options.extends = [ 'plugin:format-message/recommended' ]
+      options.customrules = null
+    } else {
+      options.extends = null
+      if (options.customrules && existsSync(options.customrules)) {
+        options.customrules = loadJSONFile(options.customrules)
+      } else {
+        options.customrules = null
+        errors.push('A valid ' + options.customrules + ' file has to be provided with extends=customrules.')
```

Comment:
This error will always be rendered `A valid null file has to be provided with extends=customrules.` Can you provide a more helpful error? It looks like you intended to include the passed path in the message, but set it to `null` first.

Reply:
Oh yea. Let me provide two different error message based on if any input was provided or none.